apps/ledger-live-desktop/src/internal/transportHandler.ts

```diff
@@ -178,6 +180,37 @@ export const transportExchangeBulkUnsubscribe = ({
   subject.next({ type: "exchangeBulkUnsubscribe", requestId });
 };
 
+const transportListenListeners = new Map<string, { unsubscribe: () => void }>();
+
+export const transportListen = ({ requestId }: { requestId: string }) => {
+  const observable = new Observable(TransportNodeHidSingleton.listen);
```

Comment:
It's probably hard in the current version, but we're coupling the specific implementation `TransportNodeHidSingleton` to this function that should probably be Transport agnostic 
It's a goal for the device team to remove this coupling ;)

But can you add a comment above the function with a warning saying we're coupling this specific transport pls ? And maybe some comment documentation to quickly describe this function pls 
(most important doc IMO: i don't see a definition of the `requestId` in this file) ?

Reply:
I can, but i have only did move existing code from listenDevice to this file btw. now at least, the transport listen is collocated with the rest where before it was a "command"

I'm going to try to write up somewhere the way the IPC Transport that was added by @Justkant last year works then. nothing is new in this PR.